bundles/org.openhab.binding.govee/README.md

```diff
@@ -20,68 +20,132 @@ While Govee provides probably more than a hundred different lights, only the fol
 
 Here is a list of the supported devices (the ones marked with * have been tested by the author)
 
-- H619Z RGBIC Pro LED Strip Lights
+- H6042 Govee TV Light Bar #2
+- H6043 Govee TV Light Bars #2
 - H6046 RGBIC TV Light Bars
 - H6047 RGBIC Gaming Light Bars with Smart Controller
-- H6061 Glide Hexa LED Panels (*)
+- H6051 Aura - Smart Table Lamp
+- H6052 Govee Table Lamp
+- H6056 H6056 Flow Plus
+- H6059 RGBWW Night Light for Kids
+- H6061 Glide Hexa LED Panels
 - H6062 Glide Wall Light
+- H6063 Gaming Wall Light
 - H6065 Glide RGBIC Y Lights
 - H6066 Glide Hexa Pro LED Panel
-- H6067 Glide Triangle Light Panels (*)
-- H6072 RGBICWW Corner Floor Lamp (*)
-- H6076 RGBICW Smart Corner Floor Lamp (*)
+- H6067 Glide Triangle Light Panels
+- H606A Glide Hexa Light Panel Ultra
+- H6072 RGBICWW Corner Floor Lamp
 - H6073 LED Floor Lamp
+- H6076 RGBICW Smart Corner Floor Lamp
 - H6078 Cylinder Floor Lamp
+- H607C Floor Lamp #2
 - H6087 RGBIC Smart Wall Sconces
-- H6173 RGBIC Outdoor Strip Lights
-- H619A RGBIC Strip Lights With Protective Coating 5M
-- H619B RGBIC LED Strip Lights With Protective Coating
-- H619C LED Strip Lights With Protective Coating
-- H619D RGBIC PRO LED Strip Lights
-- H619E RGBIC LED Strip Lights With Protective Coating
-- H61A0 RGBIC Neon Rope Light 1M
-- H61A1 RGBIC Neon Rope Light 2M
-- H61A2 RGBIC Neon Rope Light 5M
-- H61A3 RGBIC Neon Rope Light
-- H61C5 RGBIC LED Neon Rope Lights for Desks (*)
-- H61D3 Neon Rope Light 2 3M (*)
-- H61D5 Neon Rope Light 2 5M (*)
-- H61A5 Neon LED Strip Light 10
-- H61A8Neon Neon Rope Light 10
-- H618A RGBIC Basic LED Strip Lights 5M
-- H618C RGBIC Basic LED Strip Lights 5M
-- H6117 Dream Color LED Strip Light 10M
-- H6159 RGB Light Strip (*)
-- H615E LED Strip Lights 30M
-- H6163 Dreamcolor LED Strip Light 5M
+- H6088 RGBIC Cube Wall Sconces
+- H608A String Downlights 5M
+- H608B String Downlights 3M
+- H608C String Downlights 2M
+- H608D String Downlights 10M
+- H60A0 Ceiling Light
+- H60A1 Smart Ceiling Light
```

Comment:
I would add a (*) to it, meaning that has been confirmed to work (in this case by Felix)

Reply:
Ok. Done.